ci/docker/runtime_functions.sh

```diff
@@ -542,14 +543,15 @@ build_ubuntu_gpu_mkldnn() {
     build_ccache_wrappers
 
     make  \
-        DEV=1                         \
-        ENABLE_TESTCOVERAGE=1         \
-        USE_CPP_PACKAGE=1             \
-        USE_BLAS=openblas             \
-        USE_MKLDNN=1                  \
-        USE_CUDA=1                    \
-        USE_CUDA_PATH=/usr/local/cuda \
-        USE_CUDNN=1                   \
+        DEV=1                                                \
+        ENABLE_TESTCOVERAGE=1                                \
+        USE_CPP_PACKAGE=1                                    \
+        USE_BLAS=openblas                                    \
+        USE_MKLDNN=1                                         \
+        USE_CUDA=1                                           \
+        USE_CUDA_PATH=/usr/local/cuda                        \
+        USE_CUDNN=1                                          \
+        CUDA_ARCH="-gencode arch=compute_52,code=compute_52" \
```

Comment:
our most used architecture is Pascal / Volta I guess, correct me if I am wrong.
Should we use arch=compute_60 and then use gen_code for 52, 53 for compatibility?

Reply:
This section is only applicable to CI, it won't affect are users.  The archs we need to support here are those we use in CI (sm52 for the g3s, sm70 for the p3s).